manifests/cluster-authentication-operator_05_deploy.yaml

```diff
@@ -37,6 +37,8 @@ spec:
           value: quay.io/openshift/origin-hypershift:v4.0
         - name: OPERATOR_IMAGE
           value: docker.io/openshift/origin-cluster-authentication-operator:v4.0  # I believe this is supposed to exist
+        - name: RELEASE_VERSION
```

Comment:
@stlaz I held this because I feel like this does not match what I would expect (but if I mistaken just unhold and let it merge).

I was expecting version mapping logic like so:

1. https://github.com/openshift/cluster-kube-apiserver-operator/blob/b32c76452f9bada5c1d0af9c96e0f49ca99a58f7/manifests/0000_10_kube-apiserver-operator_07_clusteroperator.yaml
1. https://github.com/openshift/cluster-kube-apiserver-operator/blob/b32c76452f9bada5c1d0af9c96e0f49ca99a58f7/manifests/0000_10_kube-apiserver-operator_03_version-mapping.yaml  (maps 0.0.1-snapshot to version value which is CVO replaced)
1. https://github.com/openshift/cluster-kube-apiserver-operator/blob/b23a18bc7e6e2fe1cf24c3ac91af1d359393d434/manifests/0000_10_kube-apiserver-operator_06_deployment.yaml#L44-L47
1. https://github.com/openshift/cluster-kube-apiserver-operator/blob/b32c76452f9bada5c1d0af9c96e0f49ca99a58f7/manifests/image-references

Example mapping in https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_cluster-kube-apiserver-operator/336/pull-ci-openshift-cluster-kube-apiserver-operator-master-e2e-aws/1664/artifacts/e2e-aws/configmaps.json

```json
{
    "apiVersion": "v1",
    "data": {
        "0.0.1-2019-03-14-113227": "registry.svc.ci.openshift.org/ci-op-wkn8rctb/stable@sha256:9b33a26c9d30fc0197f9cb1b8c6232c91c0be804a996e243a9a72b35603bbc56",
        "1.12.4": "registry.svc.ci.openshift.org/ci-op-wkn8rctb/stable@sha256:664ce7fe47042701cb3abfc98f922c47e672e4dc91d210180443ee3b69a69835"
    },
    "kind": "ConfigMap",
    "metadata": {
        "creationTimestamp": "2019-03-14T11:43:07Z",
        "name": "version-mapping",
        "namespace": "openshift-kube-apiserver-operator",
        "resourceVersion": "619",
        "selfLink": "/api/v1/namespaces/openshift-kube-apiserver-operator/configmaps/version-mapping",
        "uid": "5626f5b9-464e-11e9-9ef9-12d77ec16334"
    }
}
```

Reply:
See https://github.com/openshift/cluster-kube-apiserver-operator/pull/264/commits/135672d1e4fb6aef393b455308882bb80b23dd15 and https://github.com/openshift/cluster-kube-apiserver-operator/pull/308/files

Reply:
Hmm maybe we only need something like https://github.com/openshift/cluster-openshift-apiserver-operator/pull/161 and https://github.com/openshift/cluster-openshift-apiserver-operator/blob/b09ad22f97971dfec023890d100f0938dd9c3255/pkg/operator/workloadcontroller/workload_controller_openshiftapiserver_v311_00.go#L208-L223 (based on that code maybe https://github.com/openshift/cluster-authentication-operator/pull/94#discussion_r265727535 is invalid)

xref https://github.com/openshift/cluster-image-registry-operator/pull/219 and https://github.com/openshift/cluster-image-registry-operator/pull/225